app/views/decidim/vocdoni/admin/steps/index.html.erb

```diff
@@ -32,21 +33,15 @@
 </div>
 
 <% if @form %>
-  <%= decidim_form_for(@form, url: election_step_path(election, current_step), method: :patch, html: { class: "form step #{current_step}" }) do |f| %>
+  <%= decidim_form_for(@form, url: election_step_path(election, current_step), method: :patch, html: { id: "new_election_status_", class: "form step #{current_step}" }) do |f| %>
```

Comment:
As this ID is only used in the JavaScript, I think it's better to use a data attribute for this selector, as that's the [current convention recommendation in the Decidim docs](https://docs.decidim.org/en/develop/develop/guide_conventions.html#_javascript):

> Do not use CSS classes to bind JavaScript selectors or Ruby specs. Try to use an id (with the js- preffix) or a data-attribute as selector, in order to shrink the couplish between changing technologies.

(I would not change what we have already, but I'd try to not introduce new ones from now on)

Reply:
ok